engines/sci/graphics/text32.cpp

```diff
@@ -340,10 +344,19 @@ void GfxText32::drawTextBox() {
 		uint length = getLongest(&nextCharIndex, textRectWidth);
 		int16 textWidth = getTextWidth(charIndex, length);
 
-		if (_alignment == kTextAlignCenter) {
-			_drawPosition.x += (textRectWidth - textWidth) / 2;
-		} else if (_alignment == kTextAlignRight) {
-			_drawPosition.x += textRectWidth - textWidth;
+		if (g_sci->getLanguage() != Common::HE_ISR) {  // i.e. regular Left To Right direction
```

Comment:
It would be better if this check was wrapped into a separate function, e.g. isRTLLanguage(), as we might support other RTL languages in the future

Reply:
Agreed. I have even already implemented `isLanguageLTR` (I prefer positive checks rather than `if not something`.
I will update his after PR #2217 will be accepted, along with other required changes.

Reply:
Now #2217 is accepted. Please, update this PR, so we could merge it.

Reply:
@sev- , @bluegr , @criezy , a reminder to check my updates.
@sluicebox , you might want to review this.
Thanks.